checkstyle-tester/diff.groovy

```diff
@@ -103,19 +109,20 @@ def areValidCliOptions(cliOptions) {
     return valid
 }
 
-def isValidCheckstyleConfigsCombination(config, baseConfig, patchConfig) {
+def isValidCheckstyleConfigsCombination(config, baseConfig, patchConfig, patchOnly) {
     def valid = true
-    if (config && patchConfig
+    if (baseConfig && !patchConfig
+            || config && patchConfig
             || config && baseConfig
             || config && patchConfig && baseConfig) {
```

Comment:
unrelated to changes but how can this or statement be hit while the others aren't?
Whenever this line is `true` previous 2 lines are `true` as well.